public/js/data.js

```diff
@@ -822,7 +822,10 @@ createSubCircuitPrompt = function(scope = globalScope) {
     }
     if (flag) $('#insertSubcircuitDialog').append('<p>Looks like there are no other circuits which doesn\'t have this circuit as a dependency. Create a new one!</p>')
     $('#insertSubcircuitDialog').dialog({
-        width: "auto",
+        maxHeight: 350,
+        width: 250,
```

Comment:
Does the width attributes contradict each other ? I think just specifying the width would do